src/Router/test/MagicSwapV2Router.t.sol

```diff
@@ -446,34 +446,29 @@ contract MagicSwapV2RouterTest is Test {
         assertEq(address(magicSwapV2Router).balance, 0);
     }
 
-    function testAddLiquidityNFTNFT(uint256 _tokenId, uint256 _amount) public {
+    function testAddLiquidityNFTNFT(uint256 _tokenId) public {
```

Comment:
Fuzzing the `tokenId` is not ideal, because we are 
1. using consecutive ids based on the fuzzed "first" id
2. the other collection is using the same ids from the other collection

I think these restrictions kind of defeat the purpose of the fuzzing. 

If we think it's important to fuzz here (I'm not convinced), we could do the tokenIds one by one.

Wouldn't it be better to use fixed ids, but fuzz the amounts in the range of ids.length?

Reply:
Understood! We'll address test refactoring in a future PR